gen_templates.py

```diff
@@ -24,7 +24,7 @@
             "difName": "n.DIF"
         },
         {
-            "encodedAppName": "rina.apps.echotime.client-1--",
+            "encodedAppName": "traffic.generator.server-1--",
             "difName": "n.DIF"
```

Comment:
Why are you replacing rather than adding another block?
Or, better, we may replace the "rina.apps.echotime-2--" entry, I think the names is just mistaken. What do you think?